protobuf/scalapb/scalapb.proto

```diff
@@ -2,6 +2,7 @@ syntax = "proto2";
 
 package scalapb;
 
+option go_package = "scalapb.github.io/cda/grpc/trades";
```

Comment:
There are still project specific bits here (cda, trades), can you replace the entire string with the one provided?

Reply:
It's been a long day 😞 Thanks for your patience.